kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -265,8 +265,10 @@ public void testGetBestSolution() throws Exception {
         assertNotNull(score);
         assertTrue(solution.getScore().isSolutionInitialized());
         assertTrue(score.getHardScore() <= 0);
+        assertTrue(score.getHardScore() > -1000);
         // A soft score of 0 is impossible because we'll always need at least 1 computer
         assertTrue(score.getSoftScore() < 0);
+        assertTrue(score.getSoftScore() > -1000000);
```

Comment:
I wouldn't do these 2 lines: we can't really guarantee that the first initialized score will be above -1k hard and -1m soft. That's why I didn't retain them from the original test.